recipes/libftdi/1.x/patches/0001-cmake-targets.patch

```diff
@@ -0,0 +1,787 @@
+diff --git a/CMakeLists.txt b/CMakeLists.txt
+index 5aecafc..aa69995 100644
+--- a/CMakeLists.txt
++++ b/CMakeLists.txt
+@@ -1,36 +1,54 @@
+-# Project
+ project(libftdi1 C)
++
+ set(MAJOR_VERSION 1)
+ set(MINOR_VERSION 5)
+ set(PACKAGE libftdi1)
+ set(VERSION_STRING ${MAJOR_VERSION}.${MINOR_VERSION})
+ set(VERSION ${VERSION_STRING})
+-set(CMAKE_MODULE_PATH ${CMAKE_SOURCE_DIR}/cmake)
++list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/cmake")
+ 
+-# CMake
+ if("${CMAKE_BUILD_TYPE}" STREQUAL "")
+    set(CMAKE_BUILD_TYPE     RelWithDebInfo)
+ endif("${CMAKE_BUILD_TYPE}" STREQUAL "")
+ set(CMAKE_COLOR_MAKEFILE ON)
+-cmake_minimum_required(VERSION 2.6 FATAL_ERROR)
++
++cmake_minimum_required(VERSION 3.0 FATAL_ERROR)
++if(POLICY CMP0057)
++    cmake_policy(SET CMP0057 NEW)
++endif()
++if(POLICY CMP0011)
++    cmake_policy(SET CMP0011 NEW)
++endif()
++if(POLICY CMP0099)
++    cmake_policy(SET CMP0099 NEW)
++endif()
+ 
+ add_definitions(-Wall)
+ 
+ include(CMakeOptions.txt)
+ 
++set(as_subproject LibFTDI)
++macro(find_package)
++    if(NOT ${ARGV0} IN_LIST as_subproject)
++        _find_package(${ARGV})
++    endif()
++endmacro()
++
+ # Debug build
+-message("-- Build type: ${CMAKE_BUILD_TYPE}")
++message(STATUS "Build type: ${CMAKE_BUILD_TYPE}")
+ if(${CMAKE_BUILD_TYPE} STREQUAL Debug)
+    add_definitions(-DDEBUG)
+ endif(${CMAKE_BUILD_TYPE} STREQUAL Debug)
+ 
+-# find libusb
+-find_package ( USB1 REQUIRED )
+-include_directories ( ${LIBUSB_INCLUDE_DIR} )
++find_package (LibUSB REQUIRED)
++
++if (FTDIPP)
++  find_package(Boost REQUIRED)
++endif()
+ 
+-# Find Boost
+-if (FTDIPP OR BUILD_TESTS)
+-  find_package( Boost REQUIRED )
++if (APPLE)
++    find_library(APPLE_CF CoreFoundation)
++    find_library(APPLE_IOK IOKit)
+ endif()
+ 
+ # Set components
+@@ -97,7 +115,7 @@ if ( WIN32 )
+   set ( CPACK_NSIS_MODIFY_PATH             ON )
+ endif ()
+ 
+-set(CPACK_RESOURCE_FILE_LICENSE        ${CMAKE_SOURCE_DIR}/LICENSE)
++set(CPACK_RESOURCE_FILE_LICENSE        ${CMAKE_CURRENT_SOURCE_DIR}/LICENSE)
+ 
+ set(CPACK_SOURCE_GENERATOR             TGZ)
+ set(CPACK_SOURCE_IGNORE_FILES          "\\\\.git;~$;build/")
+@@ -136,7 +154,7 @@ endif ()
+ 
+ add_subdirectory(src)
+ if ( FTDIPP )
+-  project(libftdipp1 C CXX)
++  project(libftdi1 C CXX)
+   add_subdirectory(ftdipp)
+ endif ()
+ if ( PYTHON_BINDINGS )
+@@ -150,6 +168,9 @@ if ( EXAMPLES )
+ endif ()
+ add_subdirectory(packages)
+ if ( BUILD_TESTS )
++  project(libftdi1 C CXX)
++  find_package(Boost COMPONENTS unit_test_framework REQUIRED)
++  enable_testing()
+   add_subdirectory(test)
+ endif ()
```

Comment:
CCI should not be building tests!

extra

Reply:
It isn't. `BUILD_TESTS` is not defined by the cmake configuration in the conanfile.